notebooks/234-blip-visual-language-processing/234-blip-visual-language-processing.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
<p>I have and error here</p><pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[2], line 4
      2 import time
      3 from PIL import Image
----&gt; 4 from transformers import BlipProcessor, BlipForQuestionAnswering
      6 sys.path.append("../utils")
      7 from notebook_utils import download_file

ImportError: cannot import name 'BlipProcessor' from 'transformers' (c:\intel\tests\openvino_env_test\lib\site-packages\transformers\__init__.py)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
fixed